docs/tools/simpoint.md

```diff
@@ -33,7 +33,9 @@ Simpoint Checkpoint 会根据程序特性找到具有代表性的检查点。如
 !!! note
     因为 checkpoint 的原理，我们不建议在 M 态下生成 checkpoint ([Issue #54](https://github.com/OpenXiangShan/NEMU/issues/54))，如果您一定要在 M 态生成 checkpoint，我们不会提供任何支持。
 
-在 S 态或 U 态下运行 workload，比如 Linux 上运行 SPEC2006。构建的方法可以参考 [Linux Kernel for XiangShan in EMU](linux-kernel-for-xs.md) 。
+在 S 态或 U 态下运行 workload，比如 Linux 上运行 SPEC2006。构建的方法可以参考 [Linux Kernel with RISCV-pk for XiangShan in EMU](linux-kernel-for-xs.md) 或 [Linux Kernel with OpenSBI for XiangShan in EMU](opensbi-kernel-for-xs.md)
```

Comment:
If riscv-pk is deprecated for XiangShan, can we remove it?

Reply:
should we remove riscv-pk？@shinezyy

Reply:
No need to remove. I tested days ago for the latest riscv-pk. It works on XiangShan as well. I've updated the riscv-pk repo to merge upstream updates.